elastic/kibana
# [Obs AI Assistant] hide ai assistant settings card when assistant disabled

## Summary

Closes https://github.com/elastic/kibana/issues/188255

In serverless if a user doesn't have access to the the AI Assistant capability (disabled because not in space, Viewer user) we still show the settings card, but clicking on it redirects you to the Management settings with no message.  This is confusing behaviour. This change is to no longer display the card in this case.

## Testing

- Create an editor or admin user in serverless
- Go to project settings and view the "Ai assistant for Observability settings"
<img width="855" alt="Screenshot 2024-07-15 at 1 51 43 PM" src="https://github.com/user-attachments/assets/6739f718-1590-4ae5-aaa2-b65e2a000c90">
- switch to a Viewer user, the card should be gone
